hmpps-sqs-spring-boot-autoconfigure/src/main/kotlin/uk/gov/justice/hmpps/sqs/HmppsQueueService.kt

```diff
@@ -123,9 +124,8 @@ data class DlqMessage(val body: Map<String, Any>, val messageId: String)
 data class PurgeQueueRequest(val queueName: String, val sqsClient: SqsAsyncClient, val queueUrl: String)
 data class PurgeQueueResult(val messagesFoundCount: Int)
 
-internal suspend fun SqsAsyncClient.countMessagesOnQueue(queueUrl: String): Int =
+fun SqsAsyncClient.countMessagesOnQueue(queueUrl: String): CompletableFuture<Int> =
```

Comment:
`SqsAsyncClient` returns `CompletableFuture` for other methods, so it would then be weird for this one to be a `suspend` function instead.

Reply:
note I've changed to `public` so can then be referenced from other projects